service-storage/redis.ts

```diff
@@ -0,0 +1,27 @@
+import { createClient } from 'redis';
+
+const client = createClient({
+  url: process.env.REDIS_URL,
+  password: process.env.REDIS_PASSWORD,
+});
+client.on('error', (err) => console.log('Redis Client Error', err));
+
+export const getValue = async (key: string) => {
+  await client.connect();
+  const value = await client.get(key);
+  await client.disconnect();
```

Comment:
Not sure if we have pool connection here. I think it would provides significantly better performance.

Reply:
I will check this kub